code/modules/modular_computers/file_system/programs/ntnrc_client.dm

```diff
@@ -91,8 +91,7 @@
 		if("PRG_toggleadmin")
 			if(netadmin_mode)
 				netadmin_mode = FALSE
-				if(channel)
-					channel.remove_client(src) // We shouldn't be in channel's user list, but just in case...
+				channel?.add_client(src)
 				return TRUE
```

Comment:
why would we add the user when they de admin?

Reply:
so adminning removes the user, and this puts them back in the chat. otherwise, going admin mode permanently deletes the user from the chat. if you run around the station and admin the consoles, you can delete all the participants

Reply:
i believe this was an oversight but if adminning is meant to just permanently remove them i can undo this